config/locales/en.yml

```diff
@@ -715,6 +715,7 @@ en:
       add_another_feedback: Add another feedback
       add_feedback: Add feedback
       add_your_feedback: Add Your Feedback
+      admin_can_not_review_message: You cannot review this submission as you are not a coach for this student or they are not part of your assigned cohorts.
```

Comment:
@harigopal 

I am thinking to keep this message to the following:

`Admin Preview`

or 


`Admin Preview of the submission`

Reply:
See comment in second review.